src/config_handling.py

```diff
@@ -30,9 +38,13 @@ def init_config(app):
     )
 
     for config in all_configs:
-        value = os.environ.get(config, get_default_for(config))
+        if config in file_configs:
+            value = get_config_from(app, config)
+        else:
+            value = os.environ.get(config, get_default_for(config))
```

Comment:
(Marked the correct lines)
I think, you can simplify these lines just to

```suggestion
        value = get_config_from(app, config)
``` 

because you already check if `config + "_FILE"` is present and read alternatively from `os.environ`.

Reply:
get_config_from will return "openslides" for PORTs etc in dev mode then. This is not the needed behaviour.